Sources/Clappr/Classes/Base/Container.swift

```diff
@@ -92,9 +92,17 @@ open class Container: UIObject {
     func addPlugin(_ plugin: UIContainerPlugin) {
         plugins.append(plugin)
     }
+    
+    private func findPlugin(_ pluginClass: AnyClass) -> [UIContainerPlugin] {
+        return plugins.filter{ $0.isKind(of: pluginClass) }
+    }
 
     @objc open func hasPlugin(_ pluginClass: AnyClass) -> Bool {
```

Comment:
Yes, it's not really necessary right now, but as you mentioned, in the comment down below,  it might make sense for consistency matters. But yes, we could talk about its necessity at some point.